.gitignore

```diff
@@ -3,9 +3,6 @@
 #*#
 *.swp
 
-# Vendor directory
-vendor
-
```

Comment:
Checking in `vendor/` increases the repo size, but usually not by much. `vendor/` is all text, which git compresses. The benefit is that people can `clone` and `go build`. You won't have to waste time debugging issues coming from contributors with incorrect package versions. This can happen easily because `go build` searches the entire `$GOPATH`. It's really easy to accidentally build with an incorrect package version. The `vendor/` system was intended to fix this. Unfortunately, it wasn't 100% usable until `dep` came along.

Reply:
You can use `dep purge` after `dep ensure` to reduce the size of `vendor/`